docs/src/components/landing/Hero.js

```diff
@@ -43,24 +44,10 @@ export default function Hero() {
           Build scalable and secure internal tools locally. Use your own IDE, drag and drop
           pre-built components or create yours.
```

Comment:
```suggestion
          pre-built components or create your own.
```

Is it me or does "create yours" sound weird?